lib/remote_files/file.rb

```diff
@@ -16,6 +16,7 @@ def initialize(identifier, options = {})
       @configuration = RemoteFiles::CONFIGURATIONS[(options[:configuration] || :default).to_sym]
       @logger        = options[:logger]
       @populate_stored_in = options[:populate_stored_in]
+      @errors        = options[:errors] || []
```

Comment:
Should the default be `nil`? And then we just don't track errors unless the consumer of this gem supplies an errors array?

Reply:
And more specifically, this is effectively what we Arturo in the consuming service - so it becomes a no-op within this gem if the Arturo is disabled.

Reply:
Would that be safe considering we can't use the nil safe operator? Or would the `if files.error` be sufficient?

Reply:
> would the if files.error be sufficient?

Yes it would - that's basically what the safe nav operator does anyway